### PR TITLE
Allow 0-fs to start as a daemon

### PR DESCRIPTION
**Pull request by zaibon**
- also make stderr the default log output stream
- add flag to enable daemon mode and pid of the daemon

### CODE REVIEW

File: cmd/mount.go [hunk 1]
```diff
@@ -67,13 +67,40 @@ func reload(fs *g8ufs.G8ufs, cmd *Cmd) error {
...
+		WorkDir:     "./",
+		Umask:       027,
+	}
+
+	var (
+		fs  *g8ufs.G8ufs
+		err error
+	)
+
+	if cmd.Daemon {
```
Comment (muhamadazmy) [inline cmd/mount.go:87]: Is it possible that u do the mount first, before you daemonize then the child can maintain the mount while the parent exits. this way u won't have to wait in the flist module for the mount to happen.

Comment (zaibon) [reply cmd/mount.go:87]: No it doesn't work I tried that, but it seems some file descriptor or something are not properly passed to the children, I'm thinking this problems comes from fuse or something, but I didn't managed to do it like that

File: cmd/mount.go [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 import (
-	"fmt"
 	"io/ioutil"
 	"os"
 	"os/signal"
 	"path"
 	"strings"
 	"syscall"
 
+	"github.com/sevlyar/go-daemon"
```
Comment (muhamadazmy) [inline cmd/mount.go:11]: this need to be vendored

**Review by muhamadazmy — APPROVED** [review]

**Comment by despiegk** [comment]
WHY DO WE NEED THIS?
0-fs can be run underneath e.g. in corex

**Pull request merged**